tests/tests_pytorch/tuner/test_lr_finder.py

```diff
@@ -292,32 +300,10 @@ def configure_optimizers(self):
 
 def test_lr_finder_fails_fast_on_bad_config(tmpdir):
     """Test that tune fails if the model does not have a lr BEFORE running lr find."""
-    trainer = Trainer(default_root_dir=tmpdir, max_steps=2, auto_lr_find=True)
+    trainer = Trainer(default_root_dir=tmpdir, max_steps=2)
+    tuner = Tuner(trainer)
     with pytest.raises(MisconfigurationException, match="should have one of these fields"):
-        trainer.tune(BoringModel())
-
-
-def test_lr_find_with_bs_scale(tmpdir):
```

Comment:
This test is obsolete. There is no configuration anymore where both features can be active at the same time